FeedStoreChallenge/CoreDataFeedImage.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CoreDataFeedImage+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Alex Thurston on 5/7/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CoreDataFeedImage)
+class CoreDataFeedImage: NSManagedObject {
+	@NSManaged public var id: UUID
+	@NSManaged public var imageDescription: String?
+	@NSManaged public var location: String?
+	@NSManaged public var url: URL
+
+	@NSManaged public var feed: CoreDataFeed
```

Comment:
`CoreDataFeedImage` is `internal`. No need to mark properties as `public`.

Please remove `public` access modifiers.

```suggestion
	@NSManaged var id: UUID
	@NSManaged var imageDescription: String?
	@NSManaged var location: String?
	@NSManaged var url: URL

	@NSManaged var feed: CoreDataFeed
```